torch/_inductor/codegen/cuda/gemm_template.py

```diff
@@ -18,21 +21,26 @@
     Layout,
     ReinterpretView,
 )
-from ...utils import is_dynamic
+from ...utils import is_dynamic, OrderedSet
 from ...virtualized import V
 from ..common import IndentedBuffer
 from . import cutlass_utils
 from .cuda_kernel import CUDATemplateKernel
 from .cuda_template import CUTLASSTemplate
 from .cutlass_presets import gen_cutlass_presets
+from .cutlass_python_evt import CutlassEVTCodegen
+from .cutlass_utils import torch_dtype_to_cutlass_type
+
 
+GemmOperation = Any
```

Comment:
Why set to Any?

Reply:
importing types from cutlass is just really annoying (because the package may/may not exist) it's just easier to use Any here and do my best to give the reader the context. (They know what a GemmOperation is)